content/admin.py

```diff
@@ -37,8 +38,12 @@ class ImageContentAdmin(CompareVersionAdmin):
 class LatexAdmin(CompareVersionAdmin):
     """LaTeX admin
 
-    Represents the LaTeX model in the admin panel.
+    Represents the Latex model in the admin panel.
+
+    :attr LatexAdmin.fields: Including fields into the form
```

Comment:
"Including fields into the form"? What does that mean?
-> do you mean "included form fields" or something like that?